src/components/Menus/Sidebar/ProjectsPanel/ProjectsPanel.scss

```diff
@@ -47,13 +43,14 @@
     margin-bottom: $space-1;
   }
 
-  .projects-panel__save {
+  .projects-panel__save, .projects-panel__button {
     border-top: 1px solid $rpf-grey-150;
     padding: $space-1 0;
+    display: flex;
   }
 
   .projects-panel__save-button {
-    width: fill-available;
+    width: 100%;
     margin-bottom: $space-1;
```

Comment:
same as the download button to avoid this

![image](https://github.com/RaspberryPiFoundation/editor-ui/assets/74183390/0ce16267-8bb7-49e8-bd81-97dfc10f342d)

```suggestion
    margin-bottom: $space-1;
    justify-content: center;
```